hnn_core/network.py

```diff
@@ -1066,6 +1066,10 @@ def plot_cells(self, ax=None, show=True):
         """
         return plot_cells(net=self, ax=ax, show=show)
 
+    def set_cell_biophysics(self, property, cell_type=None, synapse=None,
+                            receptor=None):
```

Comment:
I think this PR should now try to bring `cell` to the `Network` class like we discussed. And this function will just set the `cell.p_secs`